src/demo/containers/playground/WidgetDemo.tsx

```diff
@@ -100,13 +100,11 @@ export const WidgetDemo: React.FunctionComponent = () => {
       <br />
       <Checkbox
         label="Initially Checked Checkbox"
-        id="ch1"
```

Comment:
Attempt to fix GH action responsible for NPM publish, GH tag, and styleguidist output.  https://github.com/Sage-Bionetworks/Synapse-React-Client/runs/3604908922?check_suite_focus=true

Reply:
This alone won't fix the automated deploy (there's other configuration we are missing), but I can take care of the rest.